platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/Browser.java

```diff
@@ -114,8 +112,7 @@ public static void launch(@NonNull final CommandLineArgs commandLineArgs, final
             return;
         }
 
-        final Path log4jPath =
-                ConfigurationHolder.getConfigData(PathsConfig.class).getLogPath();
+        final Path log4jPath = getAbsolutePath("log4j2.xml");
```

Comment:
Fine with that but please define a constant for the path

Reply:
done